kdlp.underground.software/course/fall2023/assignments/E0.md

```diff
@@ -73,7 +73,7 @@ Build, install, and boot the latest Linux kernel fresh off Linus Torvalds’ tre
     * When you power on your system, you should see a menu that looks roughly like the following:
     * As soon as it appears, hit the down arrow key to stop the countdown that will automatically boot the first selected menu entry in a couple of seconds.
 
-	<img alt="grub bootloader image from https://jfearn.fedorapeople.org/fdocs/en-US/Documentation/0.1/html/Fedora_Multiboot_Guide/GRUB-runtime.html" src="/images/grub_menu.png"></img>
+<img alt="grub bootloader image from https://jfearn.fedorapeople.org/fdocs/en-US/Documentation/0.1/html/Fedora_Multiboot_Guide/GRUB-runtime.html" src="/images/grub_menu.png"></img>
```

Comment:
just noticed that this breaks the list numbering (the subsequent bullet resets back to zero). Seems like this combo works: `    <br><img ...` although the manual br is a bit jank.